sources/en.batcave/res/source.json

```
@@ -0,0 +1,12 @@
+{
+	"info": {
+		"id": "en.batcave",
+		"name": "BatCave",
+		"version": 1,
+		"url": "https://batcave.biz/",
```

there shouldn't be a trailing slash here for the url
```suggestion
		"url": "https://batcave.biz",
```

---

Done in ca82a45